OHM-Trade-Agent-v1/app/jobs/scan_opportunities.py

```
@@ -1009,6 +1090,118 @@ def main():
+                or snapshot.symbol
+            )
+            quote_asset = str(
+                snapshot.primary_quote_currency
+                or alert.get("primary_quote_currency")
+                or (
+                    "USDT"
+                    if str(snapshot.symbol).upper().endswith("USDT")
+                    else "USD"
+                )
+            ).upper()
+            pair = to_freqtrade_pair(base_asset, quote_asset)
```

Ruff (0.16.2)

[error] Undefined name `to_freqtrade_pair`

(F821)